Update GH URL with filter to all labels (#663)
o1-labs/docs2: docs/test-world-2/bug-reporting.mdx

## Reporting Bugs

> All participants are  expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+).

Reviewer: ```suggestion
All participants are expected to raise any abnormal behavior during the protocol performance testing on the Mina [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository.
```

> - The Block Producer is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-block-producer**
> - The Load Tester is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-load-tester**

Reviewer: ```suggestion
- **Block Producers** are expected to raise any abnormal behavior during the protocol performance testing on the Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing** 
    - **Testworld-2-0-block-producer**

- **Load Testers** are expected to raise any abnormal behavior during the protocol performance testing on Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing**
    - **Testworld-2-0-load-tester**
```

> - The SNARK worker Operator is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-snark-worker**

Reviewer: ```suggestion
- **SNARK Worker Operators** are expected to raise any abnormal behavior during the protocol performance testing on Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing**
    - **Testworld-2-0-snark-worker**
```

> - The Archive Node Operator is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-archive-node**

Reviewer: ```suggestion
- **Archive Node Operators** are expected to raise any abnormal behavior during the protocol performance testing on the Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using both of these labels:

- **Testworld-2-0-protocol-performance-testing**
- **Testworld-2-0-archive-node**
```
